kubernetes/README.md

```diff
@@ -0,0 +1,62 @@
+# Kubernetes Helm Chart
+
+
+
+**Not ready for customers. Not ready for production.** 
+
+
+
+This is the Helm Chart for [Redpanda](https://vectorized.io). A Helm Chart is a collection of files used to describe a set of Kubernetes resources, and may be used to deploy and manage Redpanda in a cloud such as AWS or GCP.
+
+
+
+## Local Installation
+
+For development purposes, it can be installed locally on Linux or Mac OS.
+
+* The containers Kubernetes uses are typically Docker ones, so install that first: https://docs.docker.com/get-docker/
+
+* To create a Kubernetes cluster, you need a tool such as `kind`. The install instructions are here: https://kind.sigs.k8s.io/. (minikube is also popular but it is only good for single-node clusters.)
+
+* To install the Redpanda Helm Chart you will need `Helm`: https://helm.sh/docs/intro/install/
+
+* To run extra commands against the cluster you need `kubectl`: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+
+* Clone this repository (`deployment-automation`).
+
+  
+
+## Local Usage
+
+- Navigate to this folder (`kubernetes`).
+
+- Run `kind create cluster`
+
+- Run `helm install redpanda ./redpanda`. This will read the Helm Chart in the redpanda folder and deploy the Redpanda Docker image (the latest from Docker Hub) to pods. For now it just creates a basic 3 node Redpanda cluster.
+
+- Wait for it to rollout:
+
+  `kubectl rollout status -w statefulset/redpanda`
+
+- Create a topic `topic1`:
+
+  ```
+  kubectl run -ti --rm --restart=Never --image vectorized/redpanda rpk -- --brokers=redpanda:9092 api topic create topic1 --partitions=3 --replicas=3
```

Comment:
The rest of these commands will need to the namespace.

Reply:
Done. To avoid duplication, I took these commands out of the README. They are in the NOTES.txt, and the user will see them after running `helm install`.